codespell_lib/data/dictionary.txt

```diff
@@ -1,4 +1,5 @@
 1nd->1st
+2st->2nd
```

Comment:
Is it worth adding some 3rd variants too?

Reply:
Added 
```
3nd->3rd
3st->3rd
``